samples/04_gis_analysts_data_scientists/detecting-deforestation-using-kmeans-clustering-on-sentinel-imagery.ipynb

```diff
@@ -0,0 +1,1169 @@
+{
```

Comment:
<p>please be careful with credentials while sending pr. also provide 'home' for arcgis online and credentials for enterprise pythonapi playground </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/975/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
@rhea-j I would suggest changing the password for your portal asap

Reply:
credential changed